src/state/session/agent.ts

```diff
@@ -33,20 +33,30 @@ export async function createAgentAndResume(
     event: AtpSessionEvent,
   ) => void,
 ) {
-  const agent = new BskyAgent({service: storedAccount.service})
+  const agent = new BskyAgent({
+    service: storedAccount.service,
+    persistSession: event => {
+      const {session} = agent
+      if (!session?.did) return // @TODO SESSION - ?
+
+      onSessionChange(agent, session.did, event)
+      if (event !== 'create' && event !== 'update') {
+        addSessionErrorLog(session.did, event)
+      }
+    },
+  })
```

Comment:
`persistSession` needs to get set inside of agent constructor now. Does this make sense?

Reply:
note to self: storedAccount